README.rst

```diff
@@ -0,0 +1,96 @@
+############
+edx-learndot
+############
+
+***********
+What is it?
+***********
+
+A Django app for integrating `Open edX`_ and `Learndot`_.
+
+********
+Features
+********
+
+Right now its entire raison d'être is automatically updating Learndot
+enrolment records when a learner passes an edX course.
+
+************
+Installation
+************
+
+Prerequisites
+-------------
+
+* A working Open edX installation.
+
+Installing
+----------
+
+If you're running a Docker devstack, you can clone the repo into your
+``src`` tree and add this to your ``lms`` command in your Docker compose
+file::
+
+  pip install -e /edx/src/edx-learndot
+
+*************
+Configuration
+*************
+
+Django configuration
+--------------------
+
+Add ``edxlearndot`` to ``settings.INSTALLED_APPS``.
+
+Then you'll need to migrate your database to support storing the
+mappings between Learndot components and edX courses.
+
+Configuring access to the Learndot API
+--------------------------------------
+
+This enables your edX installation to interact with the `Learndot v2
+API`_.
+
+There are two Django settings you'll need to add:
+
+* Put your Learndot Enterprise base URL (e.g. ``https://your-sandbox.trainingrocket.com/``) in
+  ``settings.LEARNDOT_API_BASE_URL``.
+
+* Put your Learndot API key in ``settings.LEARNDOT_API_KEY``.
```

Comment:
If you include the `plugin_settings` as [suggested below](https://github.com/open-craft/edx-learndot/pull/2/files#diff-9cd30b8f8b6ed0a35c55764479e355dfR23), then it's worth mentioning that this can be done by adding `LEARNDOT_API_BASE_URL` to the `lms.env.json`, and `LEARNDOT_API_KEY` to lms.auth.json.